tests/utils/utils.go

```diff
@@ -0,0 +1,20 @@
+package utils
```

Comment:
Should this go in Tekton Controller?

Reply:
or perhaps the [other utils pkg](https://github.com/redhat-appstudio/e2e-tests/blob/7dc195834eb82673fd1a65f0c90d8ccd26752bec/pkg/utils/util.go#L127)?(we already have the `GetFailedPipelineRunDetails` there, which is used by this new function)

Reply:
@flacatus @psturc I did not put in [utils pkg](https://github.com/redhat-appstudio/e2e-tests/blob/7dc195834eb82673fd1a65f0c90d8ccd26752bec/pkg/utils/util.go) because it gives a `import cycle not allowed`

Reply:
ah, right, it needs to import the common controller which is inside utils pkg 🤦 
so would Flavius' suggestion work? I'm not in favor of adding a new utils package into the `tests` folder

Reply:
I moved it now to `pkg/utils/tekton/pipelines.go`